numpyro/distributions/continuous.py

```diff
@@ -67,6 +66,143 @@
 )
 
 
+class AsymmetricLaplace(Distribution):
+    arg_constraints = {
+        "loc": constraints.real,
+        "scale": constraints.positive,
+        "asymmetry": constraints.positive,
+    }
+    reparametrized_params = ["loc", "scale", "asymmetry"]
+    support = constraints.real
+
+    def __init__(self, asymmetry=1.0, loc=0.0, scale=1.0, validate_args=None):
```

Comment:
Can we use the same order as in Pyro, to avoid confusion?

Reply:
Sure. I wrote it this way to be consistent with `StudentT(df, loc, scale)` and with the scipy implementation `ALD(asym, loc, scale)`. I can change it back to follow Pyro if you prefer that.

Reply:
I see. Yeah, I think matching Pyro behavior is preferred. It will help users change their codes easier (and avoid hidden issues due to the wrong order)